libraries/chain/abi_serializer.cpp

```diff
@@ -223,6 +236,8 @@ namespace eosio { namespace chain {
    std::string_view abi_serializer::fundamental_type(const std::string_view& type)const {
       if( is_array(type) ) {
          return type.substr(0, type.size()-2);
+      } else if (is_szarray (type) ){
```

Comment:
Remove space `is_szarray (type)` => `is_szarry(type)`

Reply:
Can be implemented as 
```
if (auto r = get_array_type_name(type) ; !r.empty()) 
  return r;
```